doc/how-to/how-to-add-allow-list-variables.md

```diff
@@ -0,0 +1,111 @@
+# How to add `ALLOW_LIST` variables for analytics
+
+## Context
+
+For PlanX, we are using a platform called Metabase to handle analytics for how people use the services from different local authorities. In order to tell Metabase what to pick up, we allow a number of variables to be exposed, these are contained in an array called `ALLOW_LIST`. You can see these as database columns in Hasura, in the views `analytics_summary` and `submission_services_summary`.
+
+In these docs I will run through the steps for adding a new value to `ALLOW_LIST` both in the codebase and in Hasura. 
+
+## Process
+
+### Step 1 - Add variables to `ALLOW_LIST` array
+
+ You will need to add the desired variables to two places in the codebase
+  `api.planx.uk/modules/webhooks/service/analyzeSessions/operations.ts` and `editor.planx.uk/src/pages/FlowEditor/lib/analytics/provider.tsx`. 
+
+  Each file should have an array looking like the below:
+
+    ```ts
+    export const ALLOW_LIST = [
+      "application.declaration.connection",
+      "application.information.harmful",
+      "application.information.sensitive",
+      "application.type",
+    ...
+      "service.type",
+      "usedFOIYNPP",
+      "user.role",
+      *your variable*,
+    ] as const;
+    ```
+
+  Add your variable to this array.
+
+  *It is worth noting here that variables should be added in alphabetical order and relate to a respective passport variable*
```

Comment:
nit: it'd be great to add a little more context here like "Only passport variables that store non-personally-identifiable values should be added to the allow list. Sensitive or personally-identifiable data should _never_ be tracked or exposed via analytics."